package.json

```diff
@@ -58,6 +55,7 @@
     "http-status-codes": "^2.1.4",
     "ipfs-core": "^0.3.0",
     "ipfs-http-client": "^48.1.2",
+    "key-did-resolver": "^1.1.2-rc.3",
```

Comment:
I assume this pkg is not needed since it's not imported anywhere?

Reply:
as a bunch of other packages

Reply:
What do you mean?

Reply:
base64url, config.ts, colors, ejs and few others are not used. Have a separate PR in the making to drop them.

Reply:
https://github.com/ceramicnetwork/ceramic-anchor-service/pull/377 @oed